build.gradle

```diff
@@ -14,6 +14,8 @@ dependencies {
 	//to change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
 	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+	// to use Mojang's official mappings, comment the above line and uncomment below. Use at your own risk.
+	//mappings "com.mojang.minecraft:mappings:${project.minecraft_version}"
```

Comment:
This isnt how its done either

Reply:
`mappings minecraft.officialMojangMappings()`

Reply:
Fixed